elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/metadata/models/package-info.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright 2021, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+/**
+ * Models Package.
+ */
+@ReadPermission(expression = "Prefab.Role.All")
+@UpdatePermission(expression = "Prefab.Role.None")
```

Comment:
We can not test Update/Read/Create cos the `Table.java` class is abstract so the post calls won't work.